NitroxClient/Communication/DeferringPacketReceiver.cs

```diff
@@ -1,24 +1,27 @@
 ﻿using System.Collections.Generic;
 using NitroxClient.Map;
+using NitroxModel;
 using NitroxModel.DataStructures;
 using NitroxModel.Logger;
 using NitroxModel.Packets;
 
 namespace NitroxClient.Communication
 {
+    // TODO: Spinlocks don't seem to be necessary here, but I don't know for certain.
     public class DeferringPacketReceiver
     {
         private const int EXPIDITED_PACKET_PRIORITY = 999;
         private const int DEFAULT_PACKET_PRIORITY = 1;
         private const int DESIRED_CELL_MIN_LOD_FOR_ACTIONS = 1;
 
         private readonly Dictionary<AbsoluteEntityCell, Queue<Packet>> deferredPacketsByAbsoluteCell = new Dictionary<AbsoluteEntityCell, Queue<Packet>>();
-        private readonly NitroxModel.DataStructures.PriorityQueue<Packet> receivedPackets = new NitroxModel.DataStructures.PriorityQueue<Packet>();
+        private NitroxModel.DataStructures.PriorityQueue<Packet> receivedPackets;
```

Comment:
Has the removal of `readonly` (and initialisation moved to the constructor) only been done to reinitialize the PQ (rather than defining a `.Clear` method on it?).

Leave it for now - the PQ is something on our list anyway.

Reply:
Yeah - we disconnect from the service now. The "flush" may need to do more, but presently it just needs to allow the player to join a new session without issue. The fact that they still can't reconnect without closing the client is another matter -_-

Reply:
I've seen the discussion on Discord regarding removing outstanding packets in the queue and deferred structure. We can leave this for now, and add the desired functionality later on.